server/cssfile.js

```diff
@@ -35,42 +35,64 @@ CssFile.prototype.selectorFromPosition = function(line, column){
 };
 
 CssFile.prototype.setContent = function(source, callback){
-	var messages = csslint.verify(source).messages;
-	var errors = [];
-	messages.forEach(function(msg){
-		if(msg.type == 'error'){
-			errors.push(msg);
+	const errors = [];
```

Comment:
Having two concurrent promises is neat but passing one result through the function scoped var `errors` and the other through the promise return needs to be cleaned up a bit. Lets move cssparser out of a promise (as mentioned above there's not point, it's already a simple blocking call) and clean up the stylelint call to not need this errors var.